templates/claim_payment.html

```diff
@@ -3,6 +3,13 @@
 {% block content %}
     <div class="row justify-content-center">
     <div class="col-sm-6">
+        <!-- SVG Sprite-->
+        <div class="svg-icon" style="display: none;">
```

Comment:
fix up inconsistent line spacing in this block

Reply:
should be fixed